src/components/home/why-tailwind-css-section.tsx

```diff
@@ -634,7 +634,7 @@ export default function WhyTailwindCssSection() {
                       <div key={idx} className="flex items-center justify-start gap-4 p-6">
                         <div className="shrink-0">
                           <img
-                            alt=""
+                            alt={`${user.name} profile picture`}
```

Comment:
```suggestion
                            alt=""
```

We want to keep this as an empty alt text since the name is already included in the DOM anyway and for a person using a screen reader there's no added information in having them know there's a profile picture here (otherwise it'd read `${name}, ${name} profile picture` for every name